wfl/fit/ace.py

```diff
@@ -69,47 +71,40 @@ def fit(fitting_configs, ACE_name, ace_fit_params, ref_property_prefix='REF_',
     ACE_FIT_BLAS_THREADS: used by ace_fit.jl for number of threads to set for BLAS multithreading in ace_fit
     """
 
-    ace_fit_params = prepare_params(fitting_configs, ACE_name, ace_fit_params, ref_property_prefix)
+    ace_fit_params = prepare_params(fitting_configs, ace_fit_params, ref_property_prefix)
```

Comment:
I think `ACE_name` is a workflow thing, so should be added in `prepare_params`, not `run_ace_fit`, which should take `ace_fit.jl`-ready parameters. I guess this is extra minor and idea of "`run_ace_fit` shouldn't modify the parameters" is violated anyway because we do need to modify them with fitting configs filename and run_dir, if applicable, but what I suggested would be consistent with `run_gap_fit()` and `gap_multistage.fit()`

Reply:
I did it this way because I didn't like the idea of setting `ACE_fname` to something we know might be wrong in `prepare_params`, and we can't set it to the right thing because it needs to include `run_dir`, but you don't know it until you get to `run_ace_fit`.  I guess the right thing in this framework is to pass `run_dir` to `prepare_params`, because the `ACE_fname` param depends on it.  Do you think that'd be better?

Reply:
Yes, let's do that!

Reply:
OK, I'll push an update (to this PR) soon.

Reply:
Actually, I just realized (as I was fixing the failed flake8 lint CI error) that because we only wrap `run_ace_fit` in `ExPyRe`, it's easier if the config file writing happens there.  If it happens before, we need to add the configs file to the `input_files`.  Or we could move the `ExPyRe` object and wrap the `fit()` call instead.

Reply:
Let's leave the fitting configs as they are then. ExPyRe should definitely be in `run_ace_fit` - I imagine someone preparing the configs & parameters themselves and just wanting a Python/ExPyRe interface to fitting the potential.